docker/courtlistener/docker-compose.yml

```
@@ -49,6 +49,12 @@ services:
       - ../../.env.dev
     working_dir: /opt/courtlistener/cl
     command: sh -c 'npm install && exec npx webpack --progress --watch --mode=development'
+    healthcheck:
+      test: sh -c 'ps aux | grep -v grep | grep webpack >/dev/null 2>&1 && echo 0 || echo 1'
+      interval: 5s
+      retries: 5
+      start_period: 10s
+      timeout: 10s
```

Seconds count with tests and with launching a compose file. 

Why not be more aggressive since the test uses almost no resources. Perhaps:

```suggestion
    healthcheck:
      test: sh -c 'ps aux | grep -v grep | grep webpack >/dev/null 2>&1 && echo 0 || echo 1'
      interval: 1s
      retries: 45
      start_period: 1s
      timeout: 1s
```

Yours gives it 10s to initially start, then retries up to five times every five seconds. That's 10 + 5 * 5 = 35 total seconds. Mine starts after 1s, then retries every 1s up to 45 times, for a total of 46s. I can't imagine why the test would take even 1s, so I reduced the timeout to almost nothing.

Thoughts?